src/internal_rep.rs

```diff
@@ -456,6 +469,12 @@ fn get_type_annotations(
                 // Arguments are validated at function creation time
                 infos.insert(AnnotationInfo::Derive(annotation.arguments.clone()));
             }
+            "hint" => {
+                warnings.push(Warning::new("The hint annotation is not yet supported",
+                              file,
+                              annotation.name.get_range().unwrap(), // DO NOT MERGE
```

Comment:
"// DO NOT MERGE"

Reply:
Haha, that's why I put those in I guess :)